tests/unit_tests/test_wave_06.py

```diff
@@ -261,11 +293,161 @@ def test_swap_best_by_category_no_other_match_is_false():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    #assert
+    assert result == False
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_a in tai.inventory
+    assert item_b in tai.inventory
+    assert item_c in tai.inventory
+    assert item_d in jesse.inventory
+    assert item_e in jesse.inventory
+    assert item_f in jesse.inventory
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
     # Assertions should check:
     # - That result is falsy
     # - That tai and jesse's inventories are the correct length
     # - That all the correct items are in tai and jesse's inventories
+
+
+#additional tests
+def test_swap_best_by_category_no_inventory_is_false():
+    # Arrange
+    tai = Vendor(
+        inventory=[]
+    )
+
+    jesse = Vendor(
+        inventory=[]
+    )
+
+    # Act
+    result = tai.swap_best_by_category(
+        other_vendor=jesse,
+        my_priority="Electronics",
+        their_priority="Decor"
+    )
+
+    #assert
+    assert result == False
+    assert len(tai.inventory) == 0
+    assert len(jesse.inventory) == 0
+    assert not tai.inventory 
+    assert not jesse.inventory
```

Comment:
Like stated in the other comment, these assert statements don't explicitly test for anything and already implicitly checked in the asserts above.
```suggestion
```